src/lib.rs

```diff
@@ -237,6 +287,10 @@ impl<'a> EbpfVmMbuff<'a> {
     pub fn prog_exec(&self, mem: &[u8], mbuff: &[u8]) -> u64 {
         const U32MAX: u64 = u32::MAX as u64;
 
+        let prog = match self.prog { 
+            Some(prog) => prog,
+            None => panic!("No program, call prog_set()"),
```

Comment:
Suggestion: `"Error: no program set, call prog_set() to load one"`
(In any case, please prefix with `Error: ` to remain consistent with other messages)